recipes/configs/llama2/7B_lora_single_device_deep_lake_raft.yaml

```diff
@@ -0,0 +1,94 @@
+# Config for single device LoRA finetuning in lora_finetune_single_device.py
```

Comment:
I thought it would be helpful to have an example because it's not just another data set but it's another technique to fine tune with. I thought this example would make it easier to use this technique